Services/Repository/classes/class.ilRepositoryObjectPlugin.php

```diff
@@ -130,7 +130,10 @@ protected function beforeActivation()
 
 		// add rbac operations
 		// 1: edit_permissions, 2: visible, 3: read, 4:write, 6:delete
-		$ops = array(1, 2, 3, 4, 6);
+		
```

Comment:
Kriegen wir das nicht auch ohne Patch hin? Vorschlag wäre eine entsprechende Query in die afterActivation des Plugins zu stopfen.